rvic/core/read_forcing.py

```diff
@@ -298,27 +298,22 @@ def read(self, timestamp):
         # Get the liquid fluxes
         log.info('getting fluxes for %s (%s)', timestamp, self.current_tind)
         forcing = {}
-        for i, fld in enumerate(self.liq_flds):
-            self.current_fhdl.variables[fld].set_auto_maskandscale(False)
-            if i == 0:
-                forcing['LIQ'] = \
-                    (self.fld_mult[fld] *
-                     self.current_fhdl.variables[fld][self.current_tind, :, :])
-            else:
-                forcing['LIQ'] += \
-                    (self.fld_mult[fld] *
-                     self.current_fhdl.variables[fld][self.current_tind, :, :])
+        for flux, fields in (('LIQ', self.liq_flds), ('ICE', self.ice_flds)):
```

Comment:
I just meant the names - I am not advocating allowing to route more quantities, but have a 

LIQUID: <varname>

line in a configuration file. As said, maybe for later